install/db/dbmigrate_drop_table_constraints.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/***********************************************************
+ Copyright (C) 2018 Siemens AG
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ ***********************************************************/
+
+/**
+ * \file dbmigrate_drop_table_constraints.php
+ * Drop constraints from tables,
+ */
+
+/**
+ * @brief Drop errored constraints.
+ *
+ * @param DbManager $dbManager
+ * @param string $tableName
+ * @param string $constraintName
+ */
+function dropConstraints($dbManager, $tableName, $constraintName)
+{
+  $sql = "ALTER TABLE $tableName DROP CONSTRAINT IF EXISTS $constraintName";
+  $dbManager->getSingleRow($sql,array(),"dropConstraints".$constraintName);
+}
+
+$listOfConstraints = [
+  ["clearing_event", "clearing_event_rf_fk_fkey"]
+];
+
+foreach ($listOfConstraints as $listConstraintsRow) {
+  $result = $dbManager->getSingleRow("SELECT count(*) FROM information_schema.table_constraints WHERE constraint_name = $1;", array($listConstraintsRow[1]), 'checkconstraintsIfExists');
+  if($result['count'] > 0) {
```

Comment:
This will always drop the constraint `clearing_event_rf_fk_fkey` even if it would be corrected to also include candidate licenses. Maybe also check for database version and apply if version is `3.4.0` and increase afterwards.